artifactory/commands/transferfiles/utils.go

```diff
@@ -169,11 +239,11 @@ func removeDeletedChunksFromSet(deletedChunks []string, deletedChunksSet *datast
 	}
 }
 
-func handleChunksStatuses(phase *phaseBase, chunksStatus []ChunkStatus, progressbar *TransferProgressMng,
-	awaitingStatusChunksSet *datastructures.Set[string], deletedChunksSet *datastructures.Set[string],
-	timeEstMng *timeEstimationManager, errorsChannelMng *ErrorsChannelMng) bool {
+func handleChunksStatuses(phase *phaseBase, chunksStatus *UploadChunksStatusResponse, progressbar *TransferProgressMng,
```

Comment:
Let's add a description for this function.

Reply:
done